governance/openssf_scorecard_incubation_stage.md

```diff
@@ -0,0 +1,89 @@
+- [Allstar](https://github.com/ossf/allstar) is a GitHub App that continuously monitors GitHub organizations or repositories for adherence to security best practices. Allstar has since been added an OpenSSF Scorecard project.
+- [Prominent OpenSSF Scorecard Users](https://github.com/ossf/scorecard?tab=readme-ov-file#prominent-scorecard-users)
+- [4.5k stars](https://github.com/ossf/scorecard/stargazers), [~500 forks](https://github.com/ossf/scorecard/forks)
+- [GitHub dependency graph](https://github.com/ossf/scorecard/network/dependents)
+
+## Governance
+
+Project must have met publicly at least 5 times in the last quarter since becoming Sandbox
+
+- Link to public meeting notes (or ideally recordings): https://docs.google.com/document/d/1b6d3CVJLsl7YnTE7ZaZQHdkdYIvuOQ8rzAmvVdypOWM/edit?usp=sharing
+
+Projects must have documented, initial project governance
+
+- https://github.com/ossf/scorecard/blob/main/CHARTER.md
+
+Project must have defined Contributor Guide
+
+- https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md
+
+Project has attained an OpenSSF Best Practice Badge at "passing" level
+
+- https://www.bestpractices.dev/en/projects/5621
+
+Project is integrated into the OpenSSF Scorecard
+
+- https://scorecard.dev/viewer/?uri=github.com/ossf/scorecard
+
+## IP policy and licensing due diligence
+
+When contributing an existing Project to the OpenSSF, the contribution must undergo license and IP due diligence by the Linux Foundation (LF). This step is only needed for the initial donation and only applicable here, if the project intends to join the OpenSSF Incubation stage.
+
+N/A, this project has been under OpenSSF governance for multiple years
+
+## Project References
+
+The project should provide a list of existing resources with links to the repository, website, a roadmap, contributing guide, demos and walkthroughs, and any other material to showcase the existing breadth, maturity, and direction of the project.
+
+| Reference | URL |
+|---|---|
+| Repo | https://github.com/ossf/scorecard |
+| Meeting Agenda | https://docs.google.com/document/d/1b6d3CVJLsl7YnTE7ZaZQHdkdYIvuOQ8rzAmvVdypOWM/edit?usp=sharing |
+| OSSF Calendar Entry | https://github.com/ossf/scorecard?tab=readme-ov-file#join-the-scorecard-project-meeting |
+| Website | https://scorecard.dev/ |
+| Contributing guide | https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md |
+| Security.md | https://github.com/ossf/scorecard/blob/main/SECURITY.md |
+| Roadmap | https://github.com/orgs/ossf/projects/24/views/4 |
+| Demos | https://openssf.org/training/securing-projects-with-openssf-scorecard-course/ |
+| Best Practices Badge | https://www.bestpractices.dev/en/projects/5621 |
+| Scorecard integration | https://scorecard.dev/viewer/?uri=github.com/ossf/scorecard |
+| Other | N/A |
```

Comment:
should the PR title be updated to remove (Draft?)

(ignore this tagged location, just opening a conversation to block PR until marked resolved )

Reply:
Swapped the word "draft" with "add".

I've also updated the PR description as follows:

This PR is a draft of the OpenSSF Scorecard project's submission to become an OpenSSF Incubation level project.
The actual application will be submitted to https://github.com/ossf/tac.

Drafting it in our repo to:

- Allow an opportunity for @ossf/scorecard-maintainers to review the application ahead of submission
- Provide a point-in-time copy of our application within our governance docs (we'll do something similar as we work towards graduation)